python/tvm/meta_schedule/builder/local_builder.py

```diff
@@ -15,20 +15,36 @@
 # specific language governing permissions and limitations
 # under the License.
 """Local builder that compile on the local host"""
+import logging
 import os
 import tempfile
-from typing import Callable, List, Optional, Union
+from typing import Callable, Dict, List, Optional, Union
 
 from tvm._ffi import register_func
 from tvm.ir import IRModule
-from tvm.runtime import Module
+from tvm.runtime import Module, NDArray, load_param_dict, save_param_dict
 from tvm.target import Target
 
 from ...contrib.popen_pool import MapResult, PopenPoolExecutor, StatusKind
 from ..utils import cpu_count, get_global_func_with_default_on_worker
 from .builder import BuilderInput, BuilderResult, PyBuilder
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Likely we will need to append `# pylint: disable=invalid-name` to this line to suppress the linter

Reply:
Thank you for the feedback! I've just pushed the updated commit.

Reply:
Thanks! just click “resolve conversation” yourself if the comment is addressed :-)